### ISSUE TITLE
Splitting same vendor prefix

### ISSUE THREAD

**Issue by crstauf** [yuschick/stylelint-plugin-defensive-css#32]
**Describe the bug**
The error `split each vendor prefix into its own, individual rule` shows even when the vendor prefix is the same:

```css
.a video::-webkit-media-controls-panel
.b video::-webkit-media-controls-panel {
    display: none;
}
```

**To Reproduce**
1. Create CSS rule with two of the same vendor prefix
3. Run Stylelint

**Expected behavior**
No error.

**Comment by yuschick** [comment]
Yet again, another good catch. Sorry for having so many issues with this one rule. I'll get to this this week 👍

**Comment by yuschick** [comment]
This has been fixed up and published in v1.0.2 🤘
